src/main/java/edu/ucsb/ucsbcslas/controllers/OnlineOfficeHoursController.java

```diff
@@ -27,8 +39,19 @@
 
 import edu.ucsb.ucsbcslas.advice.AuthControllerAdvice;
 import edu.ucsb.ucsbcslas.entities.OnlineOfficeHours;
+
+//new
```

Comment:
This comment will lose its meaning once it is merged into main, as it is no longer "new", please remove.